sale_stock_picking_blocking/models/account_payment_term.py

```diff
@@ -0,0 +1,16 @@
+# Copyright 2024 ForgeFlow S.L.
+#   (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import fields, models
+
+
+class AccountPaymentTerm(models.Model):
+    _inherit = "account.payment.term"
+
+    default_delivery_block = fields.Many2one(
```

Comment:
A Many2one field name should be suffixed by id.
```suggestion
    default_delivery_block_reason_id = fields.Many2one(
```